include/bitcoin/client/obelisk/obelisk_types.hpp

```diff
@@ -48,8 +40,10 @@ typedef std::vector<history_row> history_list;
 
 struct BCC_API stealth_row
 {
-    data_chunk ephemkey;
-    payment_address address;
+    // No sign byte in public key.
+    hash_digest ephemkey;
+    // No version byte in address.
+    short_hash address;
     hash_digest transaction_hash;
```

Comment:
OK, I see now that the address extracted from the tx can be generated as multisig in the future.

Currently in our client-side transaction deserialization we use `bc::extract(payment_address, script_type)` to determine the payment address of any output. Presumably this is the same method used to determine the short_hash of the address on the server. If so why would we not just send the version byte, is it a bandwidth optimization?